backend/messaging/wizard_bus_test.go

```diff
@@ -87,8 +86,8 @@ func TestWizardBus(t *testing.T) {
 		received++
 		messages = append(messages, m.(string))
 	}
-	assert.Equal(t, 7, len(messages))
-	assert.Equal(t, 7, received)
+	assert.Equal(t, 9, len(messages))
```

Comment:
Out of curiosity, could you explain the change in the expected message count here? Were we actually dropping messages in the test too?

Reply:
Yes we were... one of the channels didn't have enough buffer and the messages were being dropped as a result. This was being explicitly tested for.